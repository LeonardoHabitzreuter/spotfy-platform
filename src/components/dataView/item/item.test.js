import React from 'react'
import { shallow } from 'enzyme'
import DataViewItem from './index'
import Icon from '../../icon'

describe('DataViewItem component', () => {
  const wrapper = shallow(<DataViewItem id='1' />)

  test('should render properly', () => {
    expect(wrapper).toMatchSnapshot()
  })

  test('should render a generic image icon when no image was passed', () => {
    const icon = wrapper.find(Icon)

    expect(icon).not.toBeNull()
  })
})
