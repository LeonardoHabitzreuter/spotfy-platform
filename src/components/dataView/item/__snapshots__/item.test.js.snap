// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataViewItem component should render properly 1`] = `
<li
  className="list-group-item"
>
  <h3 />
  <div
    className="d-xs-flex text-sm-center text-xl-left"
  />
</li>
`;
